models/repo/mirror.go

```diff
@@ -56,21 +54,16 @@ func (m *Mirror) BeforeInsert() {
 	}
 }
 
-// AfterLoad is invoked from XORM after setting the values of all fields of this object.
-func (m *Mirror) AfterLoad(session *xorm.Session) {
-	if m == nil {
-		return
+// GetRepository returns the repository.
+func (m *Mirror) GetRepository() *Repository {
```

Comment:
Looks like we can't do that. Because the interface of `RemoteMirrorer` needs a method like that.
```go
type RemoteMirrorer interface {
	GetRepository() *Repository
	GetRemoteName() string
}
```

Reply:
haha ....